nitransforms/cli.py

```diff
@@ -0,0 +1,122 @@
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
```

Comment:
click is pretty lightweight - @effigies do you think would be some resistance in nibabel to migrate `nib-ls` and other command line scripts I might not be aware of?

Reply:
In general, nibabel has tried to avoid mandatory dependencies besides `numpy`. You might get some pushback on click, but might not, given it doesn't have any dependencies of its own.

Reply:
i've made the `main()` allow supplemental args to be passed in to `argparse.ArgumentParser.parse_args()` - hopefully that is sufficient for testing purposes